models/dvmdostem/R/write.config.dvmdostem.R

```diff
@@ -315,23 +434,47 @@ write.config.dvmdostem <- function(defaults = NULL, trait.values, settings, run.
   # TODO:
   #  [x] finish with parameter update process
   #  [x] dynamically copy parameters to right place
-  #  - dynamically copy the output_spec from insts folder to the
-  #    right place (see variable above for getting stuff from inst)
+  #  [x] dynamically copy the output_spec from insts folder to the
+  #      right place (see variable above for getting stuff from inst)
   #  - figure out how to handle the met.
   #     -> step one is symlink from raw data locations (Model install folder)
   #        into pecan run folder, maybe do this within job.sh?
 
+  # Here we set up two things: 
+  #  1) the paths to the met drivers (temperature, precip, etc)
+  #  2) paths to the other input data files that dvmdostem requires (soil maps, 
+  #     vegetation maps, topography etc).
+  # This will allow us to source the meteorology data from PEcAn (or BetyDB) and 
+  # collect the other inputs from a different location.
+
   # Met info
   met_driver_dir <- dirname(settings$run$inputs$met$path)
-  PEcAn.logger::logger.info(paste0("Using met driver path: ", met_driver_dir))
+  # Not sure what happens here if the site is selected from the
+  # map and instead of having <met><path> tags, the xml file has
+  # <met><id> tags?
```

Comment:
Within write.configs it's fine to throw an error if that happens. By this point in the workflow if the upstream code hasn't converted `<met><id>` to `<met><path>` then there's a bug